new-website/deepchem/styles/globals.css

```diff
@@ -227,3 +228,49 @@ footer {
     @apply pb-4;
   }
 }
+
+/* layers page */
```

Comment:
If you are using the exact same styling, you can just reuse the classes. No need to redeclare with a different class name.

Reply:
done.